packages/eslint-plugin/docs/rules/consistent-return.md

```diff
@@ -0,0 +1,33 @@
+---
+description: 'Require `return` statements to either always or never specify values.'
+---
+
+> 🛑 This file is source code, not the primary documentation location! 🛑
+>
+> See **https://typescript-eslint.io/rules/consistent-return** for documentation.
+
+This rule extends the base [`eslint/consistent-return`](https://eslint.org/docs/rules/consistent-return) rule.
+This version adds support for functions that return void type.
+
+<!--tabs-->
+
+#### ❌ Incorrect
```

Comment:
[Bug] The heading size is different between the two tabs, so it's not actually showing up as tabs.

![Screenshot of the consistent-return docs page showing Correct as a standalone heading](https://github.com/typescript-eslint/typescript-eslint/assets/3335181/8fd94ccc-851c-4d88-b9e0-df3ea88154ff)